nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
```

Comment:
I liked it here, as it separates general logic of `Anomaly` and a place where default and custom compute functions can be placed.